src/medley/medley.core.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
+

```

Comment:
`.csproj` is used for building/packaging the NuGet package, so I think it's necessary for the project. It could ultimately be omitted but doing so would make maintaining any sort of NuGet packaging in the future very tedious as I wrote this file manually.

Reply:
If it's for deployment, then that's fine. But we do need to take it out of the `src` directory as it'll mess with packaging and deployment of the Java version. Likewise with the `.sln` file, and the `obj` and `bin` directories. Perhaps a separate `clr` directory could be used.